samples/java-customer-registry/src/main/java/customer/CustomerValueEntityProvider.java

```diff
@@ -17,20 +17,37 @@
 package customer;
 
 import com.akkaserverless.javasdk.valueentity.ValueEntityContext;
+import com.akkaserverless.javasdk.valueentity.ValueEntityOptions;
 import com.akkaserverless.javasdk.valueentity.ValueEntityProvider;
 import com.google.protobuf.Descriptors;
 import customer.api.CustomerApi;
 
 import java.util.function.Function;
 
-// to be generated in generated-sources/src/main/java
+// FIXME to be generated in generated-sources/src/main/java
```

Comment:
Seems like generation is not used in this sample yet. I guess the protos don't have the codegen annotations yet. I'll follow up on that in separate PR.